tests/foreman/cli/test_repository.py

```diff
@@ -2586,6 +2586,59 @@ def test_positive_export_ansible_collection(
         result = default_sat.execute(f'curl {repo["published-at"]}')
         assert "available_versions" in result.stdout
 
+    @pytest.mark.tier2
+    @pytest.mark.upgrade
+    @pytest.mark.parametrize(
+        'repo_options',
+        [
+            {
+                'content-type': 'ansible_collection',
+                'url': ANSIBLE_GALAXY,
+                'ansible-collection-requirements': '{collections: [ \
+                            { name: theforeman.foreman, version: "2.1.0" }, \
+                            { name: theforeman.operations, version: "0.1.0"} ]}',
+            }
+        ],
+        ids=['ansible_galaxy'],
```

Comment:
Do you have any plan to add more parameters here ? I see the code block used multiple times, can be used as class variable use across the test.

Reply:
Probably not right now, but I'd like to leave the possibilities open for additional repo styles.